sse2neon.h

```diff
@@ -65,7 +65,7 @@
 #endif
 /* _mm_sqrt_ps and _mm_rsqrt_ps */
 #ifndef SSE2NEON_PRECISE_SQRT
-#define SSE2NEON_PRECISE_SQRT (0)
+#define SSE2NEON_PRECISE_SQRT (1)
```

Comment:
If you change the default configurations, there must be a strong reason. Think of the compatibility and consistency.